app/controllers/stafftools_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StafftoolsController < ApplicationController
```

Comment:
I set it as a route constraint here: https://github.com/education/classroom/pull/359/files#diff-21497849d8f00507c9c8dcaf6288b136R47 would it be more clear if these checks were on the controller itself?